src/test/scala/mesosphere/marathon/api/akkahttp/v2/ResponseMatchers.scala

```diff
@@ -86,4 +86,17 @@ trait ResponseMatchers { this: UnitTest =>
       HavePropertyMatchResult(matches, "podContainerSecret", Some(secret), maybeSecret)
     }
   }
+
+  /**
+    * Have property matcher for scaling policy instances.
+    * @param instances expected number of instances
+    * @return Match result
+    */
+  def scalingPolicyInstances(instances: Int) = new HavePropertyMatcher[JsValue, Option[String]] {
```

Comment:
Cool 🙂! Why is it `Option[String]` instead of `Option[Int]`, though?

Reply:
+1